gulpfile.js

```diff
@@ -25,7 +25,7 @@ gulp.task('run', function () {
   nodemon({
     script: 'src/index.js',
     ext: 'js',
-    env: { 'DEBUG': '*' },
+    /** env: { 'DEBUG': '*' }, **/
```

Comment:
If you run via `run_docker.sh` whatever value of DEBUG is set by the shell will override the Dockerfile default (see `docker-compose.yml`).  E.g. `DEBUG="*,-babel,-babel*" ./run_docker.sh` will work.
I believe that with this change you can do the same with gulp (`DEBUG="*,-babel,-babel*" gulp run`).